src/gc.c

```diff
@@ -3315,6 +3315,8 @@ static int _jl_gc_collect(jl_ptls_t ptls, jl_gc_collection_t collection)
     gc_num.sweep_time = sweep_time;
     if (sweep_full) {
         gc_num.last_full_sweep = gc_end_time;
+    } else {
```

Comment:
Put a new line between `}` and `else`. See https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#general-formatting-guidelines-for-c-code-contributions.